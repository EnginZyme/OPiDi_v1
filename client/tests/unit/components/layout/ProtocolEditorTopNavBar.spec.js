import { shallowMount } from "@vue/test-utils"
import ProtocolEditorTopNavBar from "@/components/layout/ProtocolEditorTopNavBar"


describe("shallowMounted ProtocolEditorTopNavBar", () => {
    const wrapper = shallowMount(ProtocolEditorTopNavBar)

    it("renders correctly", () => {
        expect(wrapper.element).toMatchSnapshot();
    })
})
