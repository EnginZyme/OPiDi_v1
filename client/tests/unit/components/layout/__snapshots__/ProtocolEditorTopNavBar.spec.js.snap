// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallowMounted ProtocolEditorTopNavBar renders correctly 1`] = `
<div>
  <div
    class="top_nav_bar"
  >
    <div
      class="top_nav_bar__content"
    >
      <protocolfilebanner-stub />
    </div>
  </div>
</div>
`;
